gpbft/participant.go

```diff
@@ -100,7 +102,7 @@ func (p *Participant) ValidateMessage(msg *GMessage) (valid ValidatedMessage, er
 
 	// Validate the message.
 	if err = ValidateMessage(comt.power, comt.beacon, p.host, msg); err != nil {
-		return nil, fmt.Errorf("%w: %w", ErrValidationInvalid, err)
+		return nil, xerrors.Errorf("%v: %w", err, ErrValidationInvalid)
```

Comment:
Can you please remind me what is the advantage of using `xerrors` over `fmt` in this case?

Reply:
It keeps track of line information, making debugging easier.
The key thing here is that the first `%w` was invalid.